lib/buffer.js

```diff
@@ -115,6 +115,9 @@ function assertSize(size) {
     err = new TypeError('"size" argument must be a number');
   else if (size < 0)
     err = new RangeError('"size" argument must not be negative');
+  else if (size > binding.kMaxLength)
+    err = new RangeError('"size" argument must not be larger ' +
+                         'than 0x' + binding.kMaxLength.toString(16));
```

Comment:
Why hex? It seems like having a base-10 decimal number would be easier to read/understand?